legal-api/src/legal_api/services/filings/validations/validation.py

```diff
@@ -80,13 +80,6 @@ def validate(business: Business,  # pylint: disable=too-many-branches,too-many-s
         error = PermissionService.check_user_permission(required_permission, message)
         if error:
             return error
-
-    if validate_certify_name(filing_json) and flags.is_on('enabled-deeper-permission-action'):
-        required_permission = ListActionsPermissionsAllowed.EDITABLE_CERTIFY_NAME.value
-        message = f'Permission Denied - You do not have permissions to change certified by in this filing.'
-        error = PermissionService.check_user_permission(required_permission, message)
-        if error:
-            return error
```

Comment:
I thought this would only be disabled (commented out) while a decision is pending on whether to fully remove this or not?

Reply:
Thanks.

The background on this is that the current code blocks some currently-valid scenarios (ie, where I change the certify name). But, this is pending a re-design. We may get rid of certify name altogether.

I prefer to keep this code in place, but commented out, in case we need it for the redesign.